File: app/src/main/java/com/udacity/astroapp/utils/MarsPhotoService.java
package com.udacity.astroapp.utils;

import com.udacity.astroapp.models.MarsPhotoObject;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface MarsPhotoService {

    @GET("photos?")
    Call<MarsPhotoObject> getMarsPhotoObject(@Query("earth_date") String date, @Query("api_key") String apiKey,
                                             @Query("page") String page);

}

File: app/src/main/kotlin/com/udacity/astroapp/utils/MarsPhotoService.kt
package com.udacity.astroapp.utils

import com.udacity.astroapp.models.MarsPhotoObject
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface MarsPhotoService {

    @GET("photos?")
    fun getMarsPhotoObject(
        @Query("earth_date") date: String,
        @Query("api_key") apiKey: String,
        @Query("page") page: String
    ): Call<MarsPhotoObject>
}